spec/models/manageiq/providers/google/cloud_manager/event_catcher/stream_spec.rb

```diff
@@ -3,7 +3,7 @@
 
   let(:ems)               { FactoryBot.create(:ems_google_with_project) }
   let(:subscription)      { Fog::Google::Pubsub::Subscription.new }
-  let(:pubsub_connection) { ems.connect(:service => 'pubsub') }
+  let(:pubsub_connection) { ::Fog::Google::Pubsub.new(:google_project => ems.project) }
```

Comment:
So this was calling `ems.connect` which was doing the `missing_credentials?` check (which used to be hard-coded to false)